docs/optimization/improve-parallelism/dataflow.md

```diff
@@ -1,8 +1,8 @@
 ### Enabling dataflow parallelism
 
-This guide teaches what data parallelism is and how it can improve the execution time of Concrete circuits.
+This guide explains data parallelism and how it can improve the execution time of **Concrete** circuits.
```

Comment:
should be `dataflow parallelism`, data parallelism have no meaning.

Reply:
done